### DISCUSSION TITLE
development vs danger

### DISCUSSION THREAD

**Discussion by ghost** [mikebrady/shairport-sync#1529] [Q&A]
development vs danger? What is the difference?

**Comment by mikebrady** [comment]
Danger is really volatile, really quite liable to change rapidly and to break stuff. Development is a bit less so -- it's a bit more stable.

As it happens, today most of the changes in the `danger` branch have been merged into the `development` branch. Same with NQPTP.

[Update] Sometimes the `danger` branch may just disappear.
